File: CPP/film.cpp
#include <iostream>
#include <string>
using namespace std;

class film{
    private:
    string id;
    string judul;
    string genre;
    int durasi;
    string pathGambarnya;

    public:
    film(){} //konstruktor kosong
    film(string id, string judul, string genre, int durasi, string pathGambarnya) {
        this->id = id;
        this->judul = judul;
        this->genre = genre;
        this->durasi = durasi;
        this->pathGambarnya = pathGambarnya;
    }
    string getId(){
        return id;
    }
    string getJudul(){
        return judul;
    }
    string getGenre(){
        return genre;
    }
    int getDurasi(){
        return durasi;
    }
    string getPathGambarnya(){
        return pathGambarnya;
    }

    void setId(string id) {this->id = id;}
    void setJudul(string judul) {this->judul = judul;}
    void setGenre(string genre) {this->genre = genre;}
    void setDurasi(int durasi){this->durasi = durasi;}
    void setPathGambarnya(string pathGambarnya){this->pathGambarnya = pathGambarnya;}

    ~film(){} //destruktor


};

File: JAVA/film.java
public class film {
    private String id;
    private String judul;
    private String genre;
    private int durasi;
    private String pathGambar;

    public film() {} // konstruktor kosong

    public film(String id, String judul, String genre, int durasi, String pathGambar) {
        this.id = id;
        this.judul = judul;
        this.genre = genre;
        this.durasi = durasi;
        this.pathGambar = pathGambar;
    }

    public String getId() { return id; }
    public String getJudul() { return judul; } 
    public String getGenre() { return genre; }
    public int getDurasi() { return durasi; }
    public String getPathGambar() { return pathGambar; } 

    public void setId(String id) { this.id = id; }
    public void setJudul(String judul) { this.judul = judul; }
    public void setGenre(String genre) { this.genre = genre; }
    public void setDurasi(int durasi) { this.durasi = durasi; } // Tipe data dipastikan int
    public void setPathGambar(String pathGambar) { this.pathGambar = pathGambar; }
}